### PR TITLE
Features/init

### PR DESCRIPTION
**Pull request by BAOTRAN1990**
init project and done swagger documents. You can see API document via this link: http://localhost:3000/documentation

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const Joi = require('joi');
 
 const movieController = require('../controllers/movieController');
+const movieResponseObj = require('../models/movieResponse');
+const movieDtlResponseObj = require('../models/movieDtlResponse');
```
Comment (putuyoga) [inline src/routes/index.js]: what is movieDtl stand for ? could you please elaborate it ? @BAOTRAN1990

**Review by putuyoga — COMMENTED** [review]
need some elaboration.

File: src/routes/index.js (hunk 1 above)
Comment (BAOTRAN1990) [reply src/routes/index.js]: movieDtl stand for movie detail, anyway I changed it to clearer variable name: movieDetailResponseObj

**Review by putuyoga — APPROVED** [review]
Ok, nice one.

**Pull request merged**